04.class/class.js

```diff
@@ -14,75 +15,84 @@ class Memo {
     this.files = fs.readdirSync('./memo_data', 'utf-8')
   }
 
+  getFileContent () {
+    const hash = {}
+    this.files.forEach((path) => {
+      const fileContent = fs.readFileSync(`./memo_data/${path}`, 'utf-8')
+      const text = fileContent.split(/\r\n|\r|\n/)
+      hash[text[0]] = path
+    }
+    )
```

Comment:
`((path) => {` で始まってるので、終わりはこうかな？

```suggestion
    })
```

Reply:
ここ、未対応っぽいですー

Reply:
@JunichiIto さんレビューありがとうございます！
見落としていました。申し訳ありません🙇‍♂️

こちらのコミットで修正いたしましたのでご確認よろしくお願いします。
[afa4d84](https://github.com/OdenTakashi/js-practices/pull/4/commits/afa4d840cfac3f2903fbdfd0dc65fd6f97042f90)